### ISSUE TITLE
error: no filter named 'ipwrap'

### ISSUE THREAD

**Issue by lgoldstien** [debops/ansible-tcpwrappers#3]
I'm having some issues when applying this role as a dependancy of `debops.mysql`. Getting the following error:

```
TASK: [debops.tcpwrappers | Allow access from Ansible Controller to sshd] *****
fatal: [default] => {'msg': "AnsibleError: file: /usr/local/etc/ansible/roles/debops.tcpwrappers/templates/etc/hosts.allow.d/ansible_controller.j2, line number: 7, error: no filter named 'ipwrap'", 'failed': True}
fatal: [default] => {'msg': "AnsibleError: file: /usr/local/etc/ansible/roles/debops.tcpwrappers/templates/etc/hosts.allow.d/ansible_controller.j2, line number: 7, error: no filter named 'ipwrap'", 'failed': True}

FATAL: all hosts have already failed -- aborting
```

I am running ansible 1.7.2 (latest release tag) and running my playbook against a local vagrant box using the vagrant-ansible provisioning plugin.

I am also getting errors with `debops.tcpwrappers/templates/etc/hosts.allow.d/allow.j2` on the `Generate hosts.allow entries` task, as this also contains ipwrap.

Removing the `ipwrap` filter from both files fixes the role and allows it to run in this instance. Any assistance would be appreciated.

**Comment by anneb** [comment]
Still took me some to figure this out. Steps I took:

my setup:
ansible-galaxy install debops.gitlab -p ~/ansible/roles

~/ansible/gitlab.yml
~/ansible/roles/debops.gitlab
~/ansible/roles/... #the debops.gitlab dependencies

Steps to install the filter and get it working in my setup (Ubuntu 14.04):
1. mkdir ~/ansible/filter_plugins
2. copy https://raw.githubusercontent.com/debops/debops-playbooks/master/playbooks/filter_plugins/ipaddr.py into ~/ansible/filter_plugins/ipaddr.py
3. chmod +x ~/ansible/filter_plugins/ipaddr.py
4. sudo apt-get install python-netaddr

**Comment by drybjed** [comment]
`ipaddr()` filter has been added to Ansible core, and should be released with version 1.9. Then it will "just work" after you install python-netaddr on the Ansible Controller.

**Comment by drybjed** [comment]
This issue should be resolved now since `ipaddr` filter is part of Ansible core. I'm closing this issue.
